doc/sphinx/running.rst

```diff
@@ -259,20 +261,53 @@ Usage example::
 
     system.integrator.set_steepest_descent(
         f_max=0, gamma=0.1, max_displacement=0.1)
-    system.integrator.run(20)
+    system.integrator.run(20)   # maximal number of steps
     system.integrator.set_vv()  # to switch back to velocity Verlet
 
-A wrapper function :func:`~espressomd.minimize_energy.steepest_descent` is
-available to set up the steepest descent integrator while preserving the
-original integrator. The snippet above can be rewritten to::
+Using a custom convergence criterion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``f_max`` parameter can be set to zero to prevent the integrator from
+halting when a specific force/torque is reached. The integration can then
+be carried out in a loop with a custom convergence criterion::
+
+    max_sigma = 5
```

Comment:
wouldn't you want to take the smallest LJ sigma here?

Reply:
* distance criterion was clarified
* offline discussion: energy convergence vs. minimal distance, would require touching a lot of samples and tutorials, some of which are candidate for removal once tutorial 02 part 1 and its solution scripts get replaced by the charged rod tutorial